wpeverest/user-registration
# UR-107 Feature - Combine two login options

### All Submissions:






### Changes proposed in this Pull Request:

We have separate email confirmation and separate admin approval login option which works respectively, but most of the user wants these login options work simultaneously such as first user will verify their email after email successfully verified admin has to approve user unless user can't login.

### Dependent Branches and PRs :

1. User Registration (Pro) - https://github.com/wpeverest/user-registration-pro/pull/25

### How to test the changes in this Pull Request:

1. Goto Form Builder and select Admin Approval After Email Confirmation login option.
2. Register a user and you will get an email for email verification with token.
3. Try to login without verifying the email. It should throw error notice to verify email first.
4. Now verify the email and Try to login again. It still throw an error notice to wait for admin approval.
5. Goto Admin->Users section approve or deny user. If you approve user then user can login and you denied then user can't login.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Combine two login options
